libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -130,7 +129,8 @@ typedef struct IotI2CDescriptor   * IotI2CHandle_t;
  *                          passed back to the caller in the callback.
  *
  */
```

Comment:
Same comment here

Reply:
I didn't see anything is removed here...

Reply:
sorry problem with my PR plugin